samples/dashboard/demos/climate/demo.js

```diff
@@ -26,7 +26,7 @@ let cityScope = defaultCity;
 let citySeries;
 let dataScope = defaultData;
 let navigatorSeries;
-let worldDate = new Date(Date.UTC(2010, 11, 25));
+let worldDate = 1293235200000;
```

Comment:
I think that the `Date.UTC(2010, 11, 25)` would be more readable.

Reply:
I write dates always as timestamps and divide it in my mind with 86400000. jk

It's a debug leftover. I will restore. Thanks!

Reply:
I'm really impressed :D